docs/docs/index.md

```diff
@@ -29,6 +29,24 @@ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poet
 ```powershell
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
 ```
+### dockerfile install instructions
+poetry.Dockerfile
```

Comment:
Why is all this required? Shouldn't this just be `pip install poetry`?

Reply:
What if I change the whole thing to:
```suggestion
This is not officially supported yet. Use your container/image as a virtual machine and install it using any way mentioned here and we recommend that you still use virtualenvs.
```